### ISSUE TITLE
Implement Revocation Actions

### ISSUE THREAD

**Issue by lukehinds** [keylime/rust-keylime#96]
This issue depends on #92 

When a node fails, revocation actions are sent from the verifier to all nodes running the revocation service (in our case agents). When a revocation event is received the Agent should perform local actions.

On the python version of the agent these are python scripts (prefixed with `local_action`). I expect it would be a good idea to continue to use python or some type of scripting framework. An expected question may be "why not rust?", well the user of the revocation framework is typically sys admins / devops engineers who want to program something simple such as wrap system commands or call another API. As much as I personally love rust, a low level typed language such a Rust will make it more of a challenge for users to construct their own scripts. We also need to consider folks will already have their own scripts that they use and will want to continue to use, once they change agents to the rust agent.

tl;dr revocation local action scripts should continue to be a high level scripting lang as its easier for sys admins / devops folks.

**Comment by lkatalin** [comment]
Closed in #159 

One change to note that differs from the description above is that we decided the scripts can be in a variety of languages (not just Python).
